deepmd/dpmodel/atomic_model/property_atomic_model.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+from deepmd.dpmodel.fitting.property_fitting import (
+    PropertyFittingNet,
+)
+
+from .dp_atomic_model import (
+    DPAtomicModel,
+)
+
+
+class DPPropertyAtomicModel(DPAtomicModel):
+    def __init__(self, descriptor, fitting, type_map, **kwargs):
+        assert isinstance(fitting, PropertyFittingNet)
+        super().__init__(descriptor, fitting, type_map, **kwargs)
+
+    def get_intensive(self) -> bool:
```

Comment:
you do not need it any more if you have implement the attribute in the output def